src/containers/BookForm.js

```diff
@@ -1,17 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+
+function BookForm(props) {
+  const [book, setBook] = useState({
+    title: '',
+    category: 'Biography',
+  });
+
+  const handleChange = e => {
+    switch (e.target.id) {
+      case 'title':
+        setBook({
+          ...book,
+          title: e.target.value,
+        });
+        break;
+
+      case 'category':
+        setBook({
+          ...book,
+          category: e.target.value,
+        });
+        break;
+
+      default:
+        break;
+    }
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+
+    props.createBook({
```

Comment:
- [ ] [OPTIONAL] Why don't you destruct the props at the top of the component so you can just call "createBook"?